frontends/web/src/routes/account/add/components/coin-dropdown.tsx

```diff
@@ -16,39 +17,100 @@
 
 import { useTranslation } from 'react-i18next';
 import * as backendAPI from '@/api/backend';
-import { Select } from '@/components/forms';
+import Select, { components, SingleValueProps, OptionProps, SingleValue, DropdownIndicatorProps } from 'react-select';
+import { Logo } from '@/components/icon/logo';
+import styles from './coin-dropdown.module.css';
 
 type TCoinDropDownProps = {
   onChange: (coin: backendAPI.ICoin) => void;
   supportedCoins: backendAPI.ICoin[];
   value: string;
 };
 
+type TOption = {
+  label: string;
+  value: backendAPI.ICoin['coinCode'];
+  isDisabled: boolean;
+}
+
+
+const DropdownIndicator = (props: DropdownIndicatorProps<TOption>) => {
+  return (
+    <components.DropdownIndicator {...props}>
+      <div className={styles.dropdown} />
+    </components.DropdownIndicator>
+  );
+};
+
+const SelectSingleValue = (props: SingleValueProps<TOption>) => {
+  const { value, label } = props.data;
+  return (
+    <div className={styles.singleValueContainer}>
+      <components.SingleValue {...props}>
+        <div className={styles.valueContainer}>
+          <Logo coinCode={value} alt={label} />
+          <span className={styles.selectLabelText}>{label}</span>
+        </div>
+      </components.SingleValue>
+    </div>
+  );
+};
+
+
+const SelectOption = (props: OptionProps<TOption>) => {
+  const { label, value } = props.data;
+
+  return (
+    <components.Option {...props}>
+      <div className={styles.valueContainer}>
+        <Logo coinCode={value} alt={label} />
```

Comment:
small visual nit: it would be nice if the choose text aligns with the rest of the options

<img width="423" alt="Screenshot 2024-11-12 at 07 46 28" src="https://github.com/user-attachments/assets/724d7e06-4757-4088-8844-f29510921c04">

Reply:
Makes sense to me 👍